docs/notebooks/interpret_advanced_usage.ipynb

```diff
@@ -0,0 +1,2393 @@
+{
```

Comment:
<p>The link on 'data' is broken (takes me to '<a href="https://github.com/bambinos/bambi/raw/aaf8c95650f7cf50734f11f5df36bd813b19d7b3/docs/notebooks/%22https://stats.idre.ucla.edu/stat/data/fish.csv%22" target="_blank">https://github.com/bambinos/bambi/raw/aaf8c95650f7cf50734f11f5df36bd813b19d7b3/docs/notebooks/%22https://stats.idre.ucla.edu/stat/data/fish.csv%22</a>')</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/762/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>